src/wp-includes/compat.php

```diff
@@ -380,6 +380,25 @@ function is_iterable( $var ) {
 	define( 'IMAGETYPE_WEBP', 18 );
 }
 
+if ( ! function_exists( 'str_contains' ) ) {
+	/**
+	 * Polyfill for array_key_last() function added in PHP 7.3.
+	 *
+	 * Performs a case-sensitive check indicating if needle is contained in haystack.
+	 *
+	 * @ticket 49652
+	 * @ignore
```

Comment:
- The `@ticket` annotation is not needed.
- The `@ignore` annotation is not needed.